apps/web/src/app/app/my-files/page.tsx

```
@@ -1,11 +1,17 @@
 
 export default function MyFilesPage() {
-    return (
-        <div className={"w-full h-full flex pt-8"}>
-            <FileExplorerTable data={dummyData} columns={columns}/>
-        </div>
-    );
-}
\ No newline at end of file
+    const {home} = useAuthStore();
+    if (home) {
+        redirect("/app/my-files/folder/" + home);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)